src/schemas/components/SchemaFormPropertyUnknown.vue

```diff
@@ -36,6 +36,17 @@
 
   const { property } = useSchemaProperty(() => props.property)
 
+  if (!isDefined(props.property.enum)) {
+    const valueOrDefaultValue = isDefined(props.value) ? props.value : props.property.default
+
+    const json: PrefectKindJson = {
+      __prefect_kind: 'json',
+      value: asJson(valueOrDefaultValue),
+    }
+
+    emit('update:value', json)
+  }
```

Comment:
This was happening inside the computed. Which was bad for two reaons
1. Side effects in computed properties 🤮 
2. Caused the timing to be off and would end up getting reset to `null` by a higher component